src/libuv/libuv-windows.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>dnx-libuv-windows</id>
```

Comment:
This package ID seems strange. Is this package intended to go on nuget.org or just for use in our builds?